shared/middleware/helpers/bucket_helper.rb

```diff
@@ -290,6 +290,31 @@ def delete_multiple(encrypted_channel_id, filenames)
     s3.delete_objects(bucket: @bucket, delete: {objects: objects, quiet: true})
   end
 
+  def hard_delete_channel_content(encrypted_channel_id)
+    # TODO: Handle pagination in the S3 APIs
```

Comment:
Left myself a reminder that I'd like to address once all four buckets are working and tested: The S3 APIs have limits - `list_object_versions` will return some maximum number of objects, and `delete_objects` lets you specify up to 1000 objects to delete.  Since it's possible some student has more object-versions than that in one project (most likely in the files or animations apis) we should be able to call each of those APIs multiple times and make sure we delete all of the object-versions.